src/gmt_remote.c

```diff
@@ -169,9 +169,8 @@ GMT_LOCAL struct GMT_DATA_INFO *gmtremote_data_load (struct GMT_CTRL *GMT, int *
 
 	while (fgets (line, GMT_LEN512, fp) != NULL) {
 		if (line[0] == '#') continue;	/* Skip any comments */
-		if ((nr = sscanf (line, "%s %s %s %c %lg %lg %s %lg %s %s %s %[^\n]", I[k].dir, I[k].file, I[k].inc, &I[k].reg, &I[k].scale, &I[k].offset, I[k].size, &I[k].tile_size, I[k].date, I[k].coverage, I[k].filler, I[k].remark)) != 12) {
-			/* New format - soon the only format once testing of 6.1 is over */
-			GMT_Report (GMT->parent, GMT_MSG_WARNING, "File %s should have 12 fields but only %d read for record %d - download error???\n", file, nr, k);
+		if ((nr = sscanf (line, "%s %s %s %c %lg %lg %s %lg %s %s %s %s %[^\n]", I[k].dir, I[k].file, I[k].inc, &I[k].reg, &I[k].scale, &I[k].offset, I[k].size, &I[k].tile_size, I[k].date, I[k].coverage, I[k].filler, I[k].CPT, I[k].remark)) != 13) {
+			GMT_Report (GMT->parent, GMT_MSG_WARNING, "File %s should have 13 fields but only %d read for record %d - download error???\n", file, nr, k);
```

Comment:
Like what we're doing here, we may have to update the format of `gmt_data_server.txt` someday, which will break old GMT versions. Perhaps we should say `download error or incompatible GMT versions.`. 

I'm thinking if we should/can add a line in `gmt_data_server.txt`, which gives the minimin GMT version for the GMT server.

Reply:
OK, reading your comment now.  Hm.  Since 6.1 is unreleased and 6.0.0 does not know about gmt_data_server.txt, the worst that can happen is that somebody complains and we tell them to update.

Reply:
What I mean is, after GMT 6.1.0 is released, we may find another serious bug and have to add one more column in `gmt_data_server.txt` to fix it. The new format will break GMT 6.1.0 users, while the old format will break GMT master-branch users. We have other choices and have to release GMT 6.2.0 immediately. 

Even we release GMT 6.2.0 as soon as we can, there are still many GMT 6.1.0 users. They will see the "download error????" warning but the warning is not exactly correct. 

If we add a line `6.2.0` in gmt_data_server.txt, then GMT can compare if the current GMT version is smaller than the minimum required version for the data server. If yes, then GMT can report a warning to tell users that "they have to update GMT to version x.x.x" to use remote dataset.

Reply:
OK, fine, so I will add 6.1.0 to that file and parse it.  If your version is less then update GMT.

Reply:
Great, but be careful that GMT version can be 6.1.0, 6.1.0rc1 and 6.1.0_2b42bb7_2020.06.21.

Reply:
This has to work for GMT 6.1.0 regardless of what version we are on because that is what ubuntu will offer until 2026...

Reply:
Is there any way to push them update to the latest GMT version

Reply:
You mean ubuntu?  No control, they have their own schedules.  So all the linux people taking the UNAVCO course in some weeks will have 6.0.0 at best, so we will need to have them build from git and make that super easy.